testsuite/features/secondary/min_ssh_tunnel.feature

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Register a salt system to be managed via SSH tunnel
+
+  Scenario: Delete the salt minion for SSH reverse bootstrap
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_minion"
+    When I follow "Delete System"
+    Then I should see a "Confirm System Profile Deletion" text
+    When I click on "Delete Profile"
+    And I wait until I see "has been deleted" text
+    And I cleanup minion "sle_ssh_tunnel_minion"
+    Then "sle_ssh_tunnel_minion" should not be registered
+
+  Scenario: Register this minion for push via SSH tunnel
+    Given I am authorized
+    When I go to the bootstrapping page
+    Then I should see a "Bootstrap Minions" text
+    And I enter the hostname of "sle_ssh_tunnel_minion" as "hostname"
+    And I enter "22" as "port"
+    And I enter "root" as "user"
+    And I enter "linux" as "password"
+    And I select "1-SUSE-SSH-TUNNEL-DEV-x86_64" from "activationKeys"
+    And I select the hostname of "proxy" from "proxies"
+    And I check "manageWithSSH"
+    And I click on "Bootstrap"
+    And I wait until I see "Successfully bootstrapped host!" text
+    And I wait until onboarding is completed for "sle_ssh_tunnel_minion"
+
+  Scenario: The contact method is SSH tunnel on this minion
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_minion"
+    Then I should see a "Push via SSH tunnel" text
+
+  Scenario: Install a package from this SSH tunnel minion
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_minion"
+    When I follow "Software" in the content area
+    And I follow "Install"
+    And I enter "milkyway-dummy" as the filtered package name
+    And I click on the filter button
+    And I check "milkyway-dummy" in the list
+    And I click on "Install Selected Packages"
+    And I click on "Confirm"
+    Then I should see a "1 package install has been scheduled for" text
+    When I force picking pending events on "sle_ssh_tunnel_minion" if necessary
+    Then I wait until event "Package Install/Upgrade scheduled by admin" is completed
+
+  Scenario: Remove a package from this SSH tunnel minion
+    Given I am on the Systems overview page of this "sle_ssh_tunnel_minion"
+    And I follow "Software" in the content area
+    And I follow "List / Remove"
+    And I enter "milkyway-dummy" as the filtered package name
+    And I click on the filter button
+    And I check "milkyway-dummy" in the list
+    And I click on "Remove Packages"
+    And I click on "Confirm"
+    Then I should see a "1 package removal has been scheduled" text
+    When I force picking pending events on "sle_ssh_tunnel_minion" if necessary
+    Then I wait until event "Package Removal scheduled by admin" is completed
+
+  Scenario: Run a remote command on this SSH reverse minion
```

Comment:
No need to fix.. but just so you are aware, maybe here we should also say SSH tunnel minion